concourse/enumerator.py

```diff
@@ -267,15 +267,23 @@ def __init__(self, repository_url:str, cfg_set):
         self.cfg_set = not_none(cfg_set)
         concourse_cfg = cfg_set.concourse()
         job_mapping_set = cfg_set.job_mapping(concourse_cfg.job_mapping_cfg_name())
-
-        # hack: use first mapping that matches
+        github_cfg = github_cfg_for_hostname(
```

Comment:
this will not work. do not lookup githubcfg from hostname. use the github-cfgs from the job-mappings